Author: Denis Briand <debian@denis-briand.fr>
Forwarded: https://midnight-commander.org/ticket/3839
Description: fix some spelling errors in manpage.

--- a/doc/man/mc.1.in.orig	2017-07-09 16:43:05.624760216 +0200
+++ b/doc/man/mc.1.in	2017-07-09 16:43:46.028758517 +0200
@@ -1440,7 +1440,7 @@
 to "..".
 .PP
 The 'Enable ignore directories' checkbox and input field below it
-allow to set up the list of directories that should be skip during the search
+allow one to set up the list of directories that should be skip during the search
 files (for example, you may want to avoid searches on a CD\-ROM or on a NFS
 directory that is mounted across a slow link). List components must be separated
 with a colon, here is an example:
@@ -2003,10 +2003,10 @@
 of screen above panels. Enabled by default.
 .PP
 .I Command prompt.
-If enabled, command line is avalable. Enabled by default.
+If enabled, command line is available. Enabled by default.
 .PP
 .I Keybar visible.
-If enabled, 10 lables associated with F1\-F10 keys are located at the bottom
+If enabled, 10 labels associated with F1\-F10 keys are located at the bottom
 row of screen. Enabled by default.
 .PP
 .I Hintbar visible.
